convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -60,29 +44,20 @@ def run(self):
 
         super(Convert2rhelLatest, self).run()
 
-        repo_dir = tempfile.mkdtemp(prefix="convert2rhel_repo.", dir=utils.TMP_DIR)
-        repo_path = os.path.join(repo_dir, "convert2rhel.repo")
-        utils.store_content_to_file(filename=repo_path, content=CONVERT2RHEL_REPO_CONTENT)
+        repofile_path = self._download_convert2rhel_repofile()
+        if not repofile_path:
+            return
 
         cmd = [
             "repoquery",
-            "--disablerepo=*",
-            "--enablerepo=convert2rhel",
```

Comment:
FYI for reviewers: The temp directory used for the convert2rhel repofile is created only for the
repofile and nothing else. Because we tell repoquery to look only at that
directory we don't need to specify a repoid to use - repoquery will simply use
the repoid in the downloaded repofile.